### ISSUE TITLE
Janky on Mobile Sites 

### ISSUE THREAD

**Issue by d4nte46** [simeydotme/pokemon-cards-css#35]
The cards seems to be all janky on mobile sites :/ 

https://github.com/simeydotme/pokemon-cards-css/assets/62292852/f2d4263a-5325-4c21-b3ae-f8d311930163

**Comment by simeydotme** [comment]
thanks @d4nte46  for bringing it to attention!
I assuming you're on Safari? do you know which version? I had it working OK before, but looks like maybe Safari pushed another update which broke it. This is pretty cutting edge css, so I expect there to be x-browser quirks. Sorry it's not a great experience for you! 

If I was to implement this in a production app, I'd probably turn off the shiny effects for Mobile (maybe keep the Glare).

**Comment by d4nte46** [comment]
Yeah sure, the safari is for iOS 15.6.1. I did a quick google search and it said usually the iOS and safari versions are same.
